tests/js/spec/views/organizationIntegrations/integrationRow.spec.jsx

```diff
@@ -0,0 +1,218 @@
+      expect(wrapper.find('PublishStatus').exists()).toEqual(false);
+      expect(wrapper.find('StyledLink').exists()).toEqual(false);
+      expect(wrapper).toMatchSnapshot();
+    });
+  });
+  describe('Plugin', function() {
+    it('has been installed (1 project)', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
+          type="plugin"
+          slug="twilio"
+          displayName="Twilio (SMS) "
+          status="Installed"
+          publishStatus="published"
+          configurations={1}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual('twilio');
+      expect(wrapper.find('IntegrationName').props().children).toEqual('Twilio (SMS) ');
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/plugins/twilio/`
+      );
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Installed');
+      expect(wrapper.find('PublishStatus').exists()).toEqual(false);
+      expect(wrapper.find('StyledLink').props().children).toEqual('1 Configuration');
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('has been installed (3 projects)', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
+          type="plugin"
+          slug="twilio"
+          displayName="Twilio (SMS) "
+          status="Installed"
+          publishStatus="published"
+          configurations={3}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual('twilio');
+      expect(wrapper.find('IntegrationName').props().children).toEqual('Twilio (SMS) ');
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/plugins/twilio/`
+      );
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Installed');
+      expect(wrapper.find('PublishStatus').exists()).toEqual(false);
+      expect(wrapper.find('StyledLink').props().children).toEqual('3 Configurations');
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('has not been installed', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
+          type="plugin"
+          slug="amazon-sqs"
+          displayName="Amazon SQS"
+          status="Not Installed"
+          publishStatus="published"
+          configurations={0}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual('amazon-sqs');
+      expect(wrapper.find('IntegrationName').props().children).toEqual('Amazon SQS');
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/plugins/amazon-sqs/`
+      );
```

Comment:
These seem a bit irrelevant to what you're testing (and it's been tested throughout)